# Case study: Exploratory *ACE2* analysis using multi-omic web tools



## Table of contents
 
* [Step 1 Discovering the ACE2 gene](./page.md)

* [Step 2 Genomic and sequence databases](./page2.md)

* [Step 3 searching of ACE2 Gene expression](./page3.md)

* [Step 4 searching in Cancer databases](./page4.md)

* [Step 5 Comparative databases](./page5.md)

* [Step 6 Alternative splicing databases](./page6.md)

* [Step 7 Methylation databases](./page7.md)

* [Step 8 Regulatory Databases](./page8.md)

* [Step 9 miRNA, lncRNA and circRNA Databases](./page9.md)

* [Step 10 Metabolic databases](./page10.md)

* [Step 11 Disease-specific and variant-disease association](./page11.md)

* [Step 12 Proteome databases](./page12.md)

* [Step 13 Other databases](./page13.md)

* [Complete list of databases](./page14.md)


## Introduction
<div style="text-align: justify"> This work is part of the article "Fantastic databases and where to find them." To demonstrate how the tools mentioned above work, we chose a case study on ACE2, the Angiotensin I Converting Enzyme 2, an important enzyme related to  Covid-19 and Severe Acute Respiratory Syndrome.
To evaluate the top tools provided in our study, we explore ACE2 data with the following databases and tools: </div>


## Databases used in this Case Study

Alternative splicing:
* ASPicDB - <http://srv00.recas.ba.infn.it/ASPicDB/>
* NetGene2 - <http://www.cbs.dtu.dk/services/NetGene2/>

Cancer databases:
* CCLE - <https://portals.broadinstitute.org/ccle>
* TCGA data Portal - <https://www.cancer.gov/>

Comparative databases:
* TISSUES - <https://tissues.jensenlab.org/Search>
* ToppCluster - <https://toppcluster.cchmc.org/>

Disease-specific and variant-disease association:
* MARRVEL - <http://marrvel.org/>
* Varsome - <https://varsome.com/>

Methylation databases:
* BECon - <https://redgar598.shinyapps.io/BECon/>
* DiseaseMeth - <http://bio-bigdata.hrbmu.edu.cn/diseasemeth/>

Gene expression databases:
* ESCAPE - <http://www.maayanlab.net/ESCAPE/>
* GTEX - <https://gtexportal.org/home/>

Genomic and sequence databases
* DisGeNET -  <https://www.disgenet.org/home/>
* Harmonizome - <http://amp.pharm.mssm.edu/Harmonizome/>

LncRNA and miRNA databases
* exoRBase - <http://www.exoRBase.org>
* miRDB - <http://mirdb.org/index.html>
* BONUS: Viral miRNA - <http://alk.ibms.sinica.edu.tw/cgi-bin/miRNA/miRNA.cgi>

Metabolic databases:
* CIDeR - <http://mips.helmholtz-muenchen.de/cider/>
* Human Metabolome Database - <https://hmdb.ca/>

Proteome and protein-protein interaction databases:
* PDID: Protein-Drug Interaction Database - <http://biomine.cs.vcu.edu/servers/PDID/index.php>
* The Proteome Browser - <http://proteomebrowser.org/tpb/home.jspx>

Regulatory Databases:
* Interferome - <http://interferome.its.monash.edu.au/interferome/home.jspx>
* TTSMI - <http://ttsmi.bii.a-star.edu.sg/>

Other databases:
* Semantic Body Browser - <http://sbb.cellfinder.org/>
* VDJdb (COVID-19) - <https://vdjdb.cdr3.net/>

[Next - Step 1](./page.md)
