# Case study: Exploratory ACE2 analysis using multi-omic web tools

To evaluate the top tools provided in our study, we explore ACE2 data with the following databases and tools:

Alternative splicing:
* ASPicDB - <http://srv00.recas.ba.infn.it/ASPicDB/>
* NetGene2 - <http://www.cbs.dtu.dk/services/NetGene2/>

Cancer databases:
* CCLE - <https://portals.broadinstitute.org/ccle>
* TCGA data Portal - <https://www.cancer.gov/>

Comparative databases:
* TISSUES - <https://tissues.jensenlab.org/Search>
* ToppCluster - <https://toppcluster.cchmc.org/>


Here we provide the complete list of databases:

# Alternative splicing databases

**Database** |                  **URL**                                      |  
AltExtron    | <http://bioinformatics.org.au/tools/altExtron/>               |
AS-ALPS      | <http://as-alps.nagahama-i-bio.ac.jp/index.php>               |  
ASpedia      | <http://combio.snu.ac.kr/aspedia/index.html>                  |   
ASPicDB      | <http://srv00.recas.ba.infn.it/ASPicDB/>                      |  
BrainRNA-seq | <https://www.brainrnaseq.org/>                                |  
DBASS        | <http://www.dbass.org.uk/>                                    |          
FAST DB      | <http://www.genosplice.com/alternative-splicing>              |
FLJ DB       | <http://flj.lifesciencedb.jp/top/>                            |
H-DBAS       | <http://www.h-invitational.jp/h-dbas/>                        |
HEXEvent     | <http://hexevent.mmg.uci.edu/cgi-bin/HEXEvent/HEXEventWEB.cgi>|
HOLLYWOOD    | <http://hollywood.mit.edu/hollywood/Login.php>                |
HSF          | <http://www.umd.be/HSF3/index.html>                           |
IntSplice    | <https://www.med.nagoya-u.ac.jp/neurogenetics/IntSplice/>     |
MiasDB       | <http://47.88.84.236/Miasdb/index.php>                        |
NetGene2     | <http://www.cbs.dtu.dk/services/NetGene2/>                    |
NetUTR       | <http://www.cbs.dtu.dk/services/NetUTR/>                      |
SplicePort   | <http://spliceport.cbcb.umd.edu/>                             |
SpliceProt   | <http://bioinfoteam.fiocruz.br/spliceprot/index.php>          |
TassDB       | <http://tassdb2.leibniz-fli.de/>                              |


# Cancer databases

**Database** |                  **URL**                                                     |  
arrayMap     | <https://arraymap.org/>                                                      |
CGAProject   | <https://mitelmandatabase.isb-cgc.org/>                                      |
CancerNet    | <http://bis.zju.edu.cn/CancerNet/>                                           |
CanGEM       | <http://www.cangem.org/>                                                     |
CanProVar    | <http://canprovar.zhang-lab.org/index.php>                                   |
CARGO        | <http://cargo2.bioinfo.cnio.es/>                                             |
CCLE         | <https://portals.broadinstitute.org/ccle>                                    |
CGMD         | <http://cgmd.in/>                                                            |
ChiTaRS      | <http://chitars.md.biu.ac.il/>                                               |
COSMIC       | <https://cancer.sanger.ac.uk/cosmic>                                         |
CPDB         | <https://www.library.ucdavis.edu/database/carcinogenic-potency-database-cpdb/>|
DBCAT        | <http://dbcat.cgm.ntu.edu.tw/>                                               |
DbDEMC       | <https://www.picb.ac.cn/dbDEMC/>                                             |
dbDEPC       | <https://www.scbit.org/dbdepc3/index.php>                                    |
DiseaseMeth  | <http://bio-bigdata.hrbmu.edu.cn/diseasemeth/>                               |
DriverDB     | <http://120.110.158.132:8787/driverdbv2/cancer.php>                          |
GEMiCCL      | <https://www.kobic.kr/GEMICCL/>                                              |
GeneHub-GEPIS| <http://www.cgl.ucsf.edu/Research/genentech/genehub-gepis/>                  |
GSR          | <https://popmodels.cancercontrol.cancer.gov/gsr/>                            |
HCMDB        | <http://hcmdb.i-sanger.com/index>                                            |
HNOCDB       | <http://gyanxet.com/hno.html>                                                |
ISOexpresso  | <http://wiki.tgilab.org/ISOexpresso/main.php?cat=about>                      |
MCF10A       | <https://carcinogenome.org/MCF10A/>                                          |
MERAV        | <http://merav.wi.mit.edu/>                                                   |
MethmiRbase  |<https://madlab.cpe.ku.ac.th/TR2/?itemID=108747>                              |
MGDB         |<http://bioinfo.ahu.edu.cn:8080/Melanoma/>                                    |
MiRCancer    |<http://mircancer.ecu.edu/>                                                   |
MiTranscript |<http://mitranscriptome.org/>                                                  |
MSGene       |<http://msgene.bioinfo-minzhao.org/>                                          |
MTCTScan     |<http://mulinlab.org/mtctscan>                                                |
NCI ALMANAC  |<https://dtp.cancer.gov/ncialmanac/initializePage.do>                         |
NeXtProt     |<https://www.nextprot.org/about/nextprot>                                     |
OncoDB.HCC   |<http://oncodb.hcc.ibms.sinica.edu.tw/index.htm>                              |
OncomiRdbB   |<http://tdb.ccmb.res.in/OncomiRdbB/index.htm>                                 |
Oncoyeasti   |<http://www.oncoyeasti.org/>                                                  |
Pedican      |<http://pedican.bioinfo-minzhao.org/>                                         |
Progenetix   |<https://www.progenetix.org/>                                                 |
S-MED        |<https://www.oncomir.umn.edu/SMED/basic_search.php>                           |
SelTarbase   |<http://www.seltarbase.org/>                                                  |
TANTIGEN     |<http://projects.met-hilab.org/tadb/>                                         | 
TCGA Portal  |<https://www.cancer.gov/>                                                     |
TCRex        |<https://tcrex.biodatamining.be/instructions/>                                |
UMDTP53db    |<http://www.umd.be:2072/>                                                     |




Whenever you commit to this repository, GitHub Pages will run [Jekyll](https://jekyllrb.com/) to rebuild the pages in your site, from the content in your Markdown files.

### Markdown

Markdown is a lightweight and easy-to-use syntax for styling your writing. It includes conventions for

```markdown
Syntax highlighted code block

# Header 1
## Header 2
### Header 3

- Bulleted
- List

1. Numbered
2. List

**Bold** and _Italic_ and `Code` text

[Link](url) and ![Image](src)
```

