# Step 6 Alternative splicing databases

Here we explore the alternative forms of *ACE2*, first we search in ASPic DB. The gene results can be downloaded and the following output was shown: 
<img src= "./images/aspic-1.PNG">

Aspic Results column show the gene information, Gene Structure View, Predicted Transcripts, Transcript Table, Predicted Proteins, Protein Table, Predicted Splice Sites and Intron Table.
<img src= "./images/aspic-gene-structure.PNG">
 
<img src= "./images/aspic-predicted.PNG">

The second alternative splicing database - NetGene2 Server allows the user submit of a local file with a single sequence (.FASTA archive), or submit by pasting a single sequence (.FASTA). The server runs the analysis (stimated time: 2 minutes) with the option "send the results for e-mail". The output was a extended prediction, with several options and results. This may can confuse the user (pay attention!).

********************** NetGene2 v. 2.4 ************************


The sequence: X has the following composition:

Length:	41316 nucleotides.
30.0% A, 19.0% C, 19.5% G, 31.5% T, 0.0% X, 38.4% G+C

Donor splice sites, direct strand
---------------------------------
               pos 5'->3'  phase strand  confidence  5'      exon intron    3'
                    43       2     +        0.00       TCATCGTCAG^GTAGGCCCTT
                   294       2     +        0.67       TTTTGAATAG^GTAAGTGAAG
                  1067       0     +        0.63       GACAGGAGAG^GTAAGGTTCT
                  1524       0     +        0.93       CCAAAACATG^GTGAGTTCTC H
                  2495       1     +        0.64       CTATTTCTTG^GTGAGTTACC
                  2920       1     +        0.37       TCAGACCAGG^GTAATTCTAA
                  3022       0     +        0.46       CTACACACAG^GTACAAGTCG
                  3419       1     +        0.54       GGGCTCCAAG^GTACTGGATA
                  4093       1     +        0.54       TATCTGATAG^GTATGCATGA
                  4165       1     +        0.49       GAGCCCTAGG^GTAAGGATAC
                  4497       1     +        0.60       GTGAAGCCAG^GTAGGGATTG
                  4511       0     +        0.54       GGATTGAGAA^GTGAGTCAGG
                  4520       0     +        0.37       AGTGAGTCAG^GTGGGTTGCA
                  4827       2     +        0.35       CTGAGAGGTT^GTGAGTGTTC
                  5000       2     +        0.54       GGGACTACAG^GTGCGAGCCA
                  5977       0     +        0.44       ATCAAATGAG^GTAAATTATT
                  6395       0     +        0.47       GTCCTTGGAG^GTAAGTCCTA
                  6455       1     +        0.47       AGAGGCTAAG^GTACTTGAGT
                  7405       0     +        0.83       GAGCAAACGG^GTACGTTTGT
                  7545       2     +        0.44       CTATGCCTAG^GTAAGGTGTC
                  8545       0     +        0.44       TGAAATGCAT^GTAAGAGCTT
                  8663       0     +        0.83       TTGTGCCATG^GTAAGGAATT
                 10021       0     +        0.81       CTTGAACCAG^GTAGGCTACT
                 10179       0     +        0.35       GATCAAGTCA^GTAAGTAAGT
                 10183       1     +        0.37       AAGTCAGTAA^GTAAGTCATT
                 10499       2     +        0.31       TATATGAAGA^GTATGTGGTC
                 10537       1     +        0.67       AGAGCAAATC^GTAAGTTTGC
                 12185       1     +        0.31       ATTACCAGGG^GTGAGGTGGA
                 12906       0     +        0.99       CTTTGAAGAG^GTAAGCAAGG H
                 13464       2     +        0.32       AAAAATGGAA^GTAAGACAAA
                 14120       1     +        0.34       AAGGAAGTAG^GTAACAATCT
                 14497       1     +        0.65       CATTTGCTTG^GTAAGAAGCC
                 14600       1     +        0.44       GAAAAATTCT^GTAAGTGGTT
                 14607       2     +        0.41       TCTGTAAGTG^GTTAGTAACA
                 16775       0     +        0.87       GGTGGACCAG^GTAGGAAAAA
                 17028       2     +        0.62       CCATTAAGCG^GTAAGATGTA
                 17266       1     +        0.47       CAAAACACAA^GTAAGTACAT
                 19417       0     +        0.58       CATGCTATAG^GTGAGGAAAC
                 19437       0     +        0.46       TGAGGCTCAG^GTAAATGATT
                 21029       1     +        0.49       GCGACTTCAG^GTAGTGGGGC
                 21447       2     +        0.45       GGTACATGTC^GTAAGAGAGA
                 21745       1     +        0.34       AGAATCCTAG^GTGTGTCACT
                 21780       0     +        0.44       CTTGGACAAG^GTATTTAACT
                 21969       0     +        0.81       AAGGAGGGAG^GTAAGTAAGA
                 22255       1     +        0.37       ATTGACTAGG^GTAGGGATTG
                 22500       0     +        0.54       GAAAGTATAA^GTAAGTTTTC
                 22538       1     +        0.31       ACAGTAGGCG^GTAAATATAT

**The file also continue...**

The results are divided into: Donor splice sites, direct strand, Donor splice sites, complement strand, Acceptor splice sites, direct strand, Acceptor splice sites, complement strand, CUTOFF values used for confidence, and graphical results: 

<img src= "./images/netgenee.PNG">

Note: the output - and graphical output was not explained. For more information about the output results, the user can read in the output format page <http://www.cbs.dtu.dk/services/NetGene2/output.php>


[Previous - Step 5](./page5.md) [Next - Step 7](./page7.md)
